### PR TITLE
Features/trendline

### PR DESCRIPTION
**Pull request by snack-0verflow**
# Description

- Trendline Chart (Explore / Overview mode)
- Trendline Card
- Trendline Explore Screen
- Delta tag for changes
- LAD Search
- Share Trendline screen

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevent
- [x] I have run `npm test`
- [x] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

**Review by the-kenneth — CHANGES_REQUESTED** [review]

File: src/components/Nav/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    setSelectedIndex(index);
+    onSelected(labels[index], index);
+  };
+
+  return (
+    <View style={[styles.container, props.styles]}>
+      <View>
+        <View style={[StyleSheet.absoluteFillObject]}>
+          <Animated.View
+            style={[
```
Comment (the-kenneth) [inline src/components/Nav/Tabs.tsx:40]: Doesnt need to be an array for 1 object

File: src/features/dashboard/DashboardScreen.tsx [hunk 2]
```diff
@@ -75,9 +76,19 @@ export const DashboardScreen: React.FC<Props> = ({ navigation, route }) => {
...
   }, [navigation]);
 
+  useEffect(() => {
+    (async () => {
+      const flag = await appCoordinator.shouldShowTrendLine();
+      setShowTrendline(!!localTrendline);
+    })();
+  }, [localTrendline]);
+
+  console.log(localTrendline);
```
Comment (the-kenneth) [inline src/features/dashboard/DashboardScreen.tsx:90]: Remove log

File: src/components/Nav/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <View style={[styles.container, props.styles]}>
+      <View>
+        <View style={StyleSheet.absoluteFillObject}>
+          <Animated.View
+            style={[
+              {
+                width: tabWidth,
+                transform: [{ translateX: translateValue }],
+              },
+            ]}>
```
Comment (the-kenneth) [inline src/components/Nav/Tabs.tsx:42]: This also doesn't need to be in an array.

Comment (snack-0verflow) [reply src/components/Nav/Tabs.tsx:42]: @the-kenneth the array is needed for transform style.

File: assets/charts/trendline-overview.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    //     });
+
+    //     console.log(covid_dates)
+    //     console.log(covid_dates)
+
+    //     setupChart(covid_dates, corrected_covid_positive)
+    //   })
+    //   .error(function (errors) {
+    //     console.log("errors:" + errors);
+    //   });
```
Comment (the-kenneth) [inline assets/charts/trendline-overview.html:131]: Do we need to keep this?

Comment (snack-0verflow) [reply assets/charts/trendline-overview.html:131]: no sir we dont

**Review by the-kenneth — APPROVED** [review]

**Review by the-kenneth — APPROVED** [review]

**Pull request merged**